osu.Game/GameModes/Menu/ButtonSystem.cs

```diff
@@ -285,37 +290,35 @@ public override void Load()
             {
                 base.Load();
 
-                Add(logoBounceContainer = new AutoSizeContainer());
-
-                logo = new Sprite(Game.Textures.Get(@"menu-osu"))
-                {
-                    Anchor = Anchor.Centre,
-                    Origin = Anchor.Centre,
-                };
-
-                logoBounceContainer.Add(logo);
-
                 Sprite ripple = new Sprite(Game.Textures.Get(@"menu-osu"))
                 {
                     Anchor = Anchor.Centre,
                     Origin = Anchor.Centre,
                     Alpha = 0.4f
                 };
-
-                logoBounceContainer.Add(ripple);
-
                 ripple.ScaleTo(1.1f, 500);

```

Comment:
This is causing the nullref. You cannot apply transforms before a drawable has been added to the tree. Why is ripple not part of the children add below? just define `Sprite ripple;` here and move the assignment into the children tree (and apply the transforms **after** it).